src/Comparison/IfBetweenHelper.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/*
+ * This file is part of Handlebars.php Helpers Set
+ *
+ * (c) Dmitriy Simushev <simushevds@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JustBlackBird\HandlebarsHelpers\Comparison;
+
+use Handlebars\Context;
+use Handlebars\Helper as HelperInterface;
+use Handlebars\Template;
+
+/**
+ * Conditional helper that checks if two values are equal or not.
+ *
+ * Example of usage:
+ * ```handlebars
+ *   {{#ifBetween variable min max}}
+ *     The first argument is between min and max
+ *   {{else}}
+ *     The first argument is not between min and max
+ *   {{/ifBetween}}
+ * ```
+ *
+ * @author Jesse Weigert <jesse.weigert@accretivetg.com>
+ */
+class IfBetweenHelper implements HelperInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function execute(Template $template, Context $context, $args, $source)
+    {
+        $parsed_args = $template->parseArguments($args);
+        if (count($parsed_args) != 3) {
+            throw new \InvalidArgumentException(
+                '"ifBetween" helper expects exactly three arguments.'
+            );
+        }
+
+        $variable = $context->get($parsed_args[0]);
+        $min = $context->get($parsed_args[1]);
+        $max = $context->get($parsed_args[2]);
+
+        $condition = ($variable >= $min && $variable < $max);
```

Comment:
I don't think it's easily to implement in both Handlebars.php and Handlebars.js.

What do you think about using of several helpers? Ex.:

- `ifLess` stands for `(-inf, b)` (`var < b`)
- `ifBetween` stands for `(a, b)` (`a < var < b`)
- `ifBetweenLeftClosed` stands for `[a, b)` (`a <= var < b`)
- `ifBetweenRightClosed` stands for `(a, b]` (`a < var <= b`)
- `ifBetweenClosed` stands for `[a, b]` (`a <= var <= b`)
- `ifMore` stands for `(a, +inf)` (`var > a`)

The terminology is taken from math (see [wikipedia](https://en.wikipedia.org/wiki/Interval_%28mathematics%29#Classification_of_intervals)).

With the helpers above your use case can be written as:

```handlebars
{#ifBetween var 0 100}
    $10 discount
{/ifBetween}
{#ifBetweenClosed var 100 200}
    $20 discount
{/ifBetweenClosed}
{#ifMore var 200}
    Big spender! $20 discount!
{/ifMore}
```